src/ui/MultiUserChatWindow.ts

```diff
@@ -49,7 +79,74 @@ export default class MultiUserChatWindow extends ChatWindow {
       this.contact.registerMemberHook(this.contact.getNickname(), (data) => {
          this.updatePermissionAttributes(data);
       });
-      this.updatePermissionAttributes(this.contact.getMember(this.contact.getNickname()));
+
+      this.updatePermissionAttributes(this.contact.getMember(this.contact.getNickname()));   this.element.find('div.jsxc-message-area').on('contextmenu',{ref:this},this.openMUCContextMenu);
+   }
+
+   private openMUCContextMenu(event)
```

Comment:
The context menu could also be used in different scenarios like emoticons, or similar stuff.

Reply:
i named it openMUCContextMenu in the mucuserchatwindow class because the base class has its own openContextMenu method which is calld by the muc one... the muc method extends the base method... but dit not know how to call the super method ;D